plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -785,6 +846,17 @@ public void testListTablesFromSchemaWithBigAmountOfTables()
         assertThat(getQueryRunner().execute("SHOW TABLES FROM mongodb.huge_schema").getRowCount()).isEqualTo(10_000);
     }
 
+    /**
+     * Ensures that system schemas are inaccessible
```

Comment:
There's no need to be javadoc. I would move to before the 1st `assertQueryReturnsEmptyResult`.